### ISSUE TITLE
Can you tag your latest changes?

### ISSUE THREAD

**Issue by JesseTG** [sabresaurus/PlayerPrefsEditor#4]
Hi there. I submitted this package to [OpenUPM](https://openupm.com), as seen [here](https://openupm.com/packages/com.github.sabresaurus.player-prefs-editor/). Right now, a fork is listed; once this version of the tool has a valid SemVer-based tag as described [here](https://openupm.com/docs/adding-upm-package.html#upm-package-criteria), I'll revise the site listing accordingly.

The only thing you need to do is tag your most recent stable commit with the version number given in `package.json`. Would you mind doing that when you get a chance? Thank you very much.

You can also set up [automated releases](https://medium.com/openupm/how-to-maintain-upm-package-part-2-f352fbf5f87c) for the future; once you do so, you can basically forget all about OpenUPM.

**Comment by sabresaurus** [comment]
@JesseTG Created https://github.com/sabresaurus/PlayerPrefsEditor/tree/1.2.0

Thanks for the tip on automated releases, I'll look into it

**Comment by JesseTG** [comment]
Done. I submitted the package, it should be online any minute.

**Comment by sabresaurus** [comment]
@JesseTG I can't see the package on OpenUPM, I can only see the old package which is now tracking an out of date version which has a bug in it. What can we do to get the correct package to show up in OpenUPM?

**Comment by JesseTG** [comment]
@sabresaurus Is [this](https://openupm.com/packages/com.sabresaurus.playerprefseditor/) the entry you're looking for? If so, I'll open a PR to have the outdated version removed.

**Comment by sabresaurus** [comment]
@JesseTG Yep that's the one, for some reason I couldn't find it (probably just user error).

That would be great. I'll also update the description on this repo to point to that OpenUPM link as a possible place to get it

**Comment by JesseTG** [comment]
@sabresaurus No problem, I'll file a PR to take the outdated version down now. Thank you for your work.

**Comment by sabresaurus** [comment]
Remaining work on this ticket is to automate releases
